golang/src/com/google/idea/blaze/golang/run/BlazeGoDebugRunner.java

```diff
@@ -87,8 +88,9 @@ protected RunContentDescriptor doExecute(
         .startSession(
             environment,
             new XDebugProcessStarter() {
+              @NotNull
```

Comment:
We assume that methods and variables not marked with `@Nullable` to be not null without adding the annotation.